src/index.ts

```diff
@@ -624,3 +621,70 @@ function newEventDispatcher(sdkKey: string): EventDispatcher {
     typeof window !== 'undefined' ? new BrowserNetworkStatusListener() : emptyNetworkStatusListener;
   return newDefaultEventDispatcher(eventQueue, networkStatusListener, sdkKey);
 }
+
+/**
+ * Initializes the Eppo precomputed client with configuration parameters.
+ *
+ * The purpose is for use-cases where the precomputed assignments are available from an external process
+ * that can bootstrap the SDK.
+ *
+ * This method should be called once on application startup.
+ *
+ * @param config - client configuration
+ * @returns a singleton precomputed client instance
+ * @public
+ */
```

Comment:
Comment needing updating to speak to the interface?

Reply:
Great catch! I see I didn't edit it enough